tests/e2e/cypress/integration/index.spec.js

```diff
@@ -1,25 +1,34 @@
 describe("Index page spec", () => {
-    it("Gives user transit provider options", () => {
-        cy.visit("/")
+  beforeEach(() => {
+    cy.visit("/")
+  })
 
-        cy.contains("Choose your transit provider")
-          .siblings(".btn")
-          .should('not.be.empty')
-          .each(($e) => {
-              expect($e).attr("href").to.match(/\/[a-z]{3,}$/)
-          })
-    })
+  it("Gives user transit provider options", () => {
+    cy.contains("Choose your transit provider")
+      .siblings(".btn")
```

Comment:
Can we avoid the CSS selector here? Since we follow with a check of the `href`, we're already assuming `<a>` tags right? Is that assumption better/worse than the CSS class?